firestore-bigquery-export/firestore-bigquery-change-tracker/src/bigquery/index.ts

```diff
@@ -148,6 +148,20 @@ export class FirestoreBigQueryEventHistoryTracker
     return isRetryable;
   }
 
+  private async waitForInitialization(dataset, table) {
```

Comment:
Probably worth adding a code comment on this to explain the reasoning for future reference.

Reply:
Comment added.